dependencies.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
     transformedModCompileOnly(deobf('https://media.forgecdn.net/files/4127/328/XaerosWorldMap_1.14.1.22_Forge_1.7.10.jar'))
     transformedModCompileOnly("curse.maven:immersive-engineering-231951:2299019")
     transformedMod("com.github.GTNewHorizons:harvestcraft:1.0.19-GTNH:dev")
-    transformedModCompileOnly deobf('https://media.forgecdn.net/files/4091/561/LOTRMod+v36.15.jar')
+    transformedMod deobf('https://media.forgecdn.net/files/4091/561/LOTRMod+v36.15.jar')
```

Comment:
As this is not a mod we use in GTNH, it should probably stay `CompileOnly` to not bloat the default Hodgepodge run configuration.